src/main/java/com/monumental/services/MonumentService.java

```diff
@@ -14,20 +14,8 @@
 @Service
 public class MonumentService extends ModelService<Monument> {
 
-    /**
-     * Public constructor for MonumentService
-     * Use when NOT injecting SessionFactoryService via Spring
-     * @param sessionFactoryService - instance of SessionFactoryService to use for initialization
-     */
     public MonumentService(SessionFactoryService sessionFactoryService) {
-        this.sessionFactoryService = sessionFactoryService;
-    }
-
-    /**
-     * Public default constructor for MonumentService
-     */
-    public MonumentService() {
-
+        super(sessionFactoryService);
```

Comment:
Why do we even override the constructor here if it just calls super and returns?

Reply:
Model is an abstract class so you can't do `MonumentService monumentService = new ModelService(sessionFactoryService)`. How else would you do this?